Source/EasyNetQ/Consumer/ConsumerDispatcher.cs

```diff
@@ -66,18 +83,26 @@ public void QueueAction(Action action, bool surviveDisconnect = false)
 
         public void OnDisconnected()
         {
+            int count = 0;
+
             // throw away any queued actions. RabbitMQ will redeliver any in-flight
             // messages that have not been acked when the connection is lost.
             while (transientActions.TryTake(out _))
             {
+                ++count;
             }
+
+            if (count > 0)
+                logger.Info("Throwed away {Count} queued transient actions. RabbitMQ will redeliver any in-flight messages that have not been acked when the connection is lost.", count);
```

Comment:
Let's fix the message:
1. `Count` should be lowercased.
2. The second part is a bit confusing for me, let's drop it.

As a result, I offer to change it to `{count} queued transient actions were thrown`.

Reply:
Done